stools/clair/__init__.py

```diff
@@ -95,58 +145,78 @@ def help(retval=0):
         version()
         sys.exit(0)
 
+    if args.report is None and args.report_format is not None:
```

Comment:
We should be able to remove this if the report format isn’t an option.

Reply:
Okay

Reply:
Reverted :+1: